pyproject.toml

```diff
@@ -33,6 +33,7 @@ disable = [
 line-length = 79
 [tool.ruff.lint]
 select = [
+    "B",
```

Comment:
do we not need the flake8-bugbear package? https://anaconda.org/conda-forge/flake8-bugbear

Reply:
sorry, even if we do, I only saw it's a Draft PR :+1:

Reply:
No, ruff is integrating rules from a ton of other linting packages, so you only need ruff: https://docs.astral.sh/ruff/rules/#flake8-bugbear-b